utils/test/resource_location_test.cpp

```diff
@@ -76,16 +76,16 @@ TEST(resource_location, to_presentable_file_scheme_hostname)
 {
     if (is_windows())
     {
-        std::string expected = "\\\\hostname\\Public\\folder With spaces\\filE nAme";
+        std::string expected = "\\\\localhost\\Public\\folder With spaces\\filE nAme";
 
-        resource_location res("file://hostname/Public/folder%20With%20spaces/filE%20nAme");
+        resource_location res("file://localhost/Public/folder%20With%20spaces/filE%20nAme");
         EXPECT_EQ(res.to_presentable(), expected);
     }
     else
     {
-        std::string expected = "//hostname/home/user/folder With spaces/filE nAme";
+        std::string expected = "/localhost/home/user/folder With spaces/filE nAme";
 
-        resource_location res("file://hostname/home/user/folder%20With%20spaces/filE%20nAme");
+        resource_location res("file://localhost/home/user/folder%20With%20spaces/filE%20nAme");
```

Comment:
I think that on linux this simply should not have a path representation... while on windows I can actually open \\server\share\file, I am not aware of anything like that on linux.

Reply:
Maybe we should also handle a case when hostname is `localhost` as this should be accessible on both platforms

Reply:
As discussed